gtsam/hybrid/HybridFactorGraph.cpp

```diff
@@ -175,27 +175,30 @@ pair<GaussianMixture::shared_ptr, boost::shared_ptr<Factor>> EliminateHybrid(
   DecisionTree<Key, Pair> eliminationResults(sum, eliminate);
 
   // STEP 3: Create result
-  // TODO(Frank): auto pair = eliminationResults.unzip();
+  auto pair = unzip(eliminationResults);
+  GaussianMixture::Conditionals conditionals = pair.first;
```

Comment:
You are making copies here! Make them const&

Reply:
Ah good catch

Reply:
Forgot to say this in the unzip PR: in C++17 this is really just

`auto [a, b] = unzip(ab)`!